src/main/java/org/gridsuite/study/server/StudyConstants.java

```diff
@@ -54,6 +54,7 @@ private StudyConstants() {
     public static final String QUERY_PARAM_REPORT_SEVERITY_LEVEL = "severityLevels";
     public static final String QUERY_PARAM_REPORT_WITH_ELEMENTS = "withElements";
     public static final String QUERY_PARAM_REPORT_TASKKEY_FILTER = "taskKeyFilter";
+    public static final String QUERY_PARAM_REPORT_TASKKEY_FILTER_MATCHING_TYPE = "taskKeyFilterMatchingType";
```

Comment:
Same remark as report server
Rename the notion REPORT_TASKKEY_FILTER by REPORT_NAME_FILTER